lighthouse-core/test/computed/metrics/largest-contentful-paint-test.js

```diff
@@ -16,27 +16,40 @@ const invalidDevtoolsLog = require('../../fixtures/traces/progressive-app-m60.de
 /* eslint-env jest */
 
 describe('Metrics: LCP', () => {
-  it('should error when computing a simulated value', async () => {
+  it('should compute predicted value', async () => {
     const settings = {throttlingMethod: 'simulate'};
     const context = {settings, computedCache: new Map()};
-    const resultPromise = LargestContentfulPaint.request({trace, devtoolsLog, settings}, context);
-    await expect(resultPromise).rejects.toThrow();
+    const result = await LargestContentfulPaint.request({trace, devtoolsLog, settings}, context);
+
+    expect({
+      timing: Math.round(result.timing),
+      optimistic: Math.round(result.optimisticEstimate.timeInMs),
+      pessimistic: Math.round(result.pessimisticEstimate.timeInMs),
+    }).toMatchInlineSnapshot(`
+Object {
+  "optimistic": 17637,
+  "pessimistic": 17637,
+  "timing": 17637,
+}
+`);
   });
 
   it('should compute an observed value', async () => {
     const settings = {throttlingMethod: 'provided'};
     const context = {settings, computedCache: new Map()};
     const result = await LargestContentfulPaint.request({trace, devtoolsLog, settings}, context);
 
-    assert.equal(Math.round(result.timing), 15024);
-    assert.equal(result.timestamp, 1671236939268);
+    assert.equal(Math.round(result.timing), 1744);
```

Comment:
and here's the observed numbers right?

this would only change with the trace changing.. i do see the trace changed but viewing the diff definitely isn't straightforward.

you had a trace before where the observed LCP was 15s ?

Reply:
yeah observed.

I guess I had a really bad trace before